apps/web/src/app/(main)/dashboard/newsletters/components/NewsletterContent.tsx

```
@@ -0,0 +1,115 @@
+            return (
+              <div key={index} className="my-8">
+                <img
+                  src={item.src}
+                  alt={item.alt || ""}
+                  className="w-full rounded-lg"
+                />
+              </div>
+            );
+
+          case "list":
+            const isRightAligned = item.align === "left";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)